configs/retina/retinanet_R-101-FPN_1x.yaml

```diff
@@ -0,0 +1,48 @@
+MODEL:
+  META_ARCHITECTURE: "GeneralizedRCNN"
+  WEIGHT: "catalog://ImageNetPretrained/MSRA/R-101"
+  RPN_ONLY: True
```

Comment:
I think “PRN_ONLY” should be False, due to this option maybe affect the eval process.

Reply:
Hi, I think this does not affect anything in the evaluation. RPN_ONLY is not used, once the RETINANET_ON is set as True.

Reply:
I'm really got some errors.

Problem maybe occur on here:
https://github.com/facebookresearch/maskrcnn-benchmark/blob/0dfac376668a6618ed17c2d890c92728bb2926b6/maskrcnn_benchmark/data/datasets/evaluation/coco/coco_eval.py#L24

Reply:
Thanks for letting me know. I am on vacation now but will check it once I am back.